demo-app/src/app/components/language-selector/language-selector.component.ts

```diff
@@ -35,14 +35,7 @@ export class LanguageSelectorComponent implements OnInit, OnDestroy {
 	 * Class constructor
 	 * @param translateService - the translation service of the application
 	 */
-	public constructor(protected translateService: TranslateService) {}
-
-	/**
-	 * Component lifecycle hook
-	 */
-	public ngOnInit(): void {
-		console.log(componentName + ": controller initialized");
-
+	public constructor(protected translateService: TranslateService) {
 		this.selectedLanguage = this.translateService.currentLang;
 
 		this.languageChangeSubscription = this.translateService.onLangChange.subscribe(
```

Comment:
Small question, in the file "src/app/app.component.ts", you keep the `subscriptions` in the ngOnInit().
Here, you move them in the constructor. Why ?

Reply:
Ah, I thought it might be better to keep those subscriptions in the `ngOnInit` in the `AppComponent` because they are related to the `BreakpointObserver`... which is about the media queries stuff and maybe doing that in the constructor is too soon cause the component is not rendered yet...

But that is just a guess... I have no strong opinion about it... maybe I can simply move them to the constructor too

Reply:
@SuperITMan I've updated the `AppComponent` to move the subscriptions to the constructor and it seems to work fine. So it is now consistent with the other components 😉

Reply:
Okido 😊 
I didn't think to the `BreakpointObserver` but if it works correctly in the `constructor()`, it's fine.
I think this library is ready to be released in "1.0.0" 😃 

What do you think @nicanac @christophercr ? 😊

Reply:
Mmm not sure if we should release "1.0.0"... maybe an RC? It is just because there are 2 tasks that we can still work on for the final release: #28 and #21 

What do you guys think? Or... we could also release those in "1.1.0"...

Reply:
Yes, I think we can do an RC.

Reply:
Okido ! Let's go for the RC 😊